### ISSUE TITLE
[Helmet] Gatsby now has built-in support for modifying the document head.

### ISSUE THREAD

**Issue by coderbyheart** [distributeaid/distributeaid.org#717]
Figure out how to get rid of this warning shown when starting the dev server.

```
warn gatsby-plugin-react-helmet: Gatsby now has built-in support for modifying the document head.
Learn more at https://gatsby.dev/gatsby-head
```

**Comment by jayeshchoudhary** [comment]
I would love to work on this, @coderbyheart can you assign me this issue

**Comment by dorian-edwards** [comment]
Can I have a go at this one too?

**Comment by coderbyheart** [comment]
Sure!

**Comment by dorian-edwards** [comment]
It looks like Gatsby's built in Head API requires an export from every page that needs to update the document head since it can only be exported from a page and not a component. So the existing format wrapping everything in a Layout component doesn't directly translate. 

This seems like a substantive change and I just wanted to confirm this is okay before attempting this, or get some feedback for an alternative approach if one exists.

**Comment by coderbyheart** [comment]
Can you point to the relevant docs?

**Comment by dorian-edwards** [comment]
https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/#usage-notes
https://www.gatsbyjs.com/blog/introducing-the-gatsby-head-api/

**Comment by coderbyheart** [comment]
They also mention how to de-duplicate definitions, so ...

![mando-way-this-is-the-way](https://user-images.githubusercontent.com/188915/194486133-9b7f7b41-00c0-4cfc-8bdb-521a21c89031.gif)

**Comment by coderbyheart** [comment]
Closed via #751

**Comment by coderbyheart** [comment]
Thanks @dorian-edwards!
